jukeos/src/components/Library.js

```diff
@@ -319,61 +324,76 @@ const LibraryTesting = () => {
   // useEffect(() => { fetch Spotify data here }, [accessToken]);
 
   return (
-    <div style={{ position: 'relative' }}>
-      {/* TODO: Backend - Consider adding loading spinner/state here */}
+    <>
+      {/* Fixed-position gradient that stays at bottom of viewport */}
       <div style={{
-        backgroundImage: `url(${backgroundPng})`,
-        backgroundSize: 'cover',
-        backgroundPosition: 'center',
-        backgroundRepeat: 'no-repeat',
-        backgroundAttachment: 'fixed',
-        minHeight: '100vh',
-        padding: '20px',
-        perspective: '1000px',
-        overflow: 'hidden',
-        position: 'relative',
-        zIndex: 1
+        position: 'fixed',
+        bottom: 0,
+        left: 0,
+        right: 0,
+        width: '100%',
+        height: '180px',
+        zIndex: 999,
+        pointerEvents: 'none',
+        margin: 0,
+        padding: 0,
       }}>
+        <AnimatedBlob
+          colors={colors}
+          style={{
+            width: '100%', // Back to 100%
+            height: '100%',
+            filter: 'blur(70px)', // Back to original blur amount
+            opacity: 0.75,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            borderRadius: 0,
+          }}
+          static={false}
+        />
+      </div>
+
+      {/* Your existing content */}
+      <div style={{ position: 'relative' }}>
         <div style={{
-          position: 'fixed',
-          bottom: 0,
-          left: '50%',
-          transform: 'translateX(-50%)',
-          width: '200vw',
-          height: 'auto',
-          zIndex: 2,
-          pointerEvents: 'none'
-        }}>
-          <img 
-            src={mainGradient} 
-            alt="gradient" 
-            style={{
-              width: '62%',
-              height: 'auto',
-              display: 'block',
-              margin: '0 auto'
-            }}
-          />
-        </div>
-        <div style={{
-          paddingTop: '120px',
-          paddingBottom: '40px',
-          overflowY: 'auto',
-          height: '100vh',
+          backgroundImage: `url(${backgroundPng})`,
+          backgroundSize: 'cover',
+          backgroundPosition: 'center',
+          backgroundRepeat: 'no-repeat',
+          backgroundAttachment: 'fixed',
+          minHeight: '100vh',
+          padding: '0',
+          perspective: '1000px',
+          overflow: 'hidden',
           position: 'relative',
-          zIndex: 3
+          zIndex: 1
```

Comment:
I think something that something that should be included from the below div is the top padding. We should add some top padding here I think it makes it look better. Try it out and see what you think: 
```suggestion
          zIndex: 1,
          paddingTop: '100px'
```

Reply:
Resolved in https://github.com/tennitech/juke/pull/159/commits/48711b544934ac145f809f628c0406a914e82db8